integrations/using-integrations/aws-s3.md

```diff
@@ -12,10 +12,18 @@ my_bucket = client.integration('s3/my_data_bucket')
 
 ### Reading from AWS S3
 
-Once we've loaded a connection to our S3 bucket, we can access data by using the `.file()` method. `.file()` takes as an argument the path within your S3 bucket to the file that you'd like to access:
+Once we've loaded a connection to our S3 bucket, we can access data by using the `.file()` method. `.file()` requires the following arguments:
+
+* `filepaths`: One or more paths from which files should be read.
+* `format`: The expected table format when reading from `filepaths`. We currently support `CSV`, `Parquet`, or `JSON`.
+
+See below for example usage:
 
 ```python
-customers = my_bucket.file('data/sales/customers.csv')
+customers = my_bucket.file(filepaths='data/sales/customers.csv', format='CSV')
+
+# When retrieving multiple tables, we concatenate them into a single DataFrame.
```

Comment:
Let's explicitly specify this in the text above as well. Using `s3.file`, you can retrieve either one file or many files. If it's many files, we concat into a DF. 

We're also missing how prefix search works here, I believe.

Reply:
updated